alchemy/src/cloudflare/secrets-store.ts

```diff
@@ -135,6 +167,18 @@ const _SecretsStore = Resource(
         : Date.now();
 
     if (this.phase === "update" && storeId) {
+      const currentSecrets = props.secrets || {};
+      const previousSecrets = this.output?.secrets || {};
```

Comment:
can we instead rely on server-side state here instead of this.output?

use the list secrets api https://developers.cloudflare.com/api/resources/secrets_store/subresources/stores/subresources/secrets/methods/list/